From e74cae5030bc484b25ab777b6a2846e2804c4834 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Tue, 28 Feb 2023 18:17:27 +0100
Subject: [PATCH 283/841] feat(stm32mp25-fdts): add SD-card supplies on eval
 board

Add vmmc-supply and vqmmc-supply properties in sdmmc1 node.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 731d7b9780b00805759a443aebf932a3684a74e6 feat(stm32mp25-fdts): add SD-card supplies on eval board]
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Change-Id: I8a62f687cfc167756a29d2e7a8a46b28ca31bd87
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/293072
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Pascal PAILLET <p.paillet@st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 fdts/stm32mp257f-ev1.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/fdts/stm32mp257f-ev1.dts b/fdts/stm32mp257f-ev1.dts
index 1946c551d..ccfa9f5ac 100644
--- a/fdts/stm32mp257f-ev1.dts
+++ b/fdts/stm32mp257f-ev1.dts
@@ -205,6 +205,8 @@
 	pinctrl-0 = <&sdmmc1_b4_pins_a>;
 	st,neg-edge;
 	bus-width = <4>;
+	vmmc-supply = <&vdd_sdcard>;
+	vqmmc-supply = <&vddio1>;
 	status = "okay";
 };
 
-- 
2.47.2

